src/core/make-popup.js

```diff
@@ -75,9 +75,7 @@ const renderComponent = (params, options) => {
 
   const urlWithQueryString = appendParamsToUrl(url, queryStrings)
 
-  if (!isMobile(navigator.userAgent) && isScreenBig()) {
-    render(<Popup embedId={embedId} icon={icon} onClose={close} options={options} url={urlWithQueryString} />, domNode)
-  } else {
+  if (window.forceFullscreen || (isMobile(navigator.userAgent) && !isScreenBig())) {
```

Comment:
I'm curious, how does `window.forceFullscreen` work? I couldn't find a lot of docs on it. Or it's ours? @mathio @scarciofolomarco

Reply:
It is ours. We will set this variable in shareinguito to display "mobile" embed preview (full screen modal).

Also, I dont think we should document it in README since its a hack.